proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
```

Comment:
I don't think this is necessary. Currently, only the "node" user can use the rpc client.
When we introduce a sql rpc client, we may or may not use heartbeats. Either way, we'll change it then if needed.